docs/sce_file_contents.md

```diff
@@ -296,3 +303,161 @@ For methods that rely on the HTO data, these statistics are found in the `colDat
+| `disease_ontology_term_id` | [`MONDO`](http://obofoundry.org/ontology/mondo.html) term indicating disease type. [`PATO:0000461`](http://purl.obolibrary.org/obo/PATO_0000461) indicates normal or healthy tissue. If unavailable, `NA` is used.  |
+| `tissue_ontology_term_id`| [`UBERON`](http://obofoundry.org/ontology/uberon.html) term indicating tissue of origin. If unavailable, `NA` is used. |
+| `is_primary_data` | Set to `FALSE` for all libraries to reflect that all libraries were obtained from external investigators. Required by `CELLxGENE`.             |
+
+
+### `AnnData` gene information and metrics
+
+Gene information and metrics calculated from the RNA-seq expression data are stored as a `pandas.DataFrame` in the `.var` slot, with the Ensembl ID as the names of the rows.
+
+```python
+adata_object.var # gene metrics
+```
+
+All of the per-gene data columns included in the `rowData` of the `SingleCellExperiment` objects are present in the `.var` slot of the `AnnData` object.
+To see a full description of the included columns, see the [section on gene metrics in `Components of a SingleCellExperiment object`](#singlecellexperiment-gene-information-and-metrics).
+
+The `AnnData` object also includes the following additional gene-level metadata columns:
+
+| Column name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `is_feature_filtered` | Boolean indicating if the gene or feature is filtered out in the normalized matrix but is present in the raw matrix     |
+
+
+### `AnnData` experiment metadata
+
+Metadata associated with {ref}`data processing <processing_information:Processing information>` is included in the `.uns` slot as a list.
+
+```python
+adata_object.uns # experiment metadata
+```
+
+All of the object metadata included in `SingleCellExperiment` objects are present in the `.uns` slot of the `AnnData` object.
+To see a full description of the included columns, see the [section on experiment metadata in `Components of a SingleCellExperiment object`](#singlecellexperiment-experiment-metadata).
+The only exception is that the `AnnData` object _does not_ contain the `sample_metadata` item in the `.uns` slot.
+Instead, the contents of the `sample_metadata` data frame are stored in the cell-level metadata (`.obs`).
+
+The `AnnData` object also includes the following additional items in the `.uns` slot:
+
+| Item name   | Contents                                                         |
+| ------------- | ---------------------------------------------------------------- |
+| `schema_version` | CZI schema version used for `AnnData` formatting, `3.0.0` |
```

Comment:
can't hurt?

```suggestion
| `schema_version` | CZI schema version used for `AnnData` formatting, [`3.0.0`](https://github.com/chanzuckerberg/single-cell-curation/blob/main/schema/3.0.0/schema.md) |
```

Reply:
I'd actually just eliminate the version number here, since that is the value inside.
```suggestion
| `schema_version` | CZI schema version used for `AnnData` formatting |
```